ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -195,8 +195,9 @@ def multiply(self, y, name=None):
     def isfinite(self, name=None):
         return ivy.isfinite(self._ivy_array)
 
-    def all(self, axis=None, keepdim=False, name=None):
-        return ivy.all(self.ivy_array, axis=axis, keepdims=keepdim)
+    @with_supported_dtypes({"2.5.0 and below": ("float16", "bfloat16")}, "paddle")
+    def all(self, axis=None, keepdim=False, dtype=None, name=None):
+        return ivy.all(self.ivy_array, axis=axis, keepdims=keepdim, dtype=dtype)

```

Comment:
Are changes to this function necessary for your function? If not, it'd be better to revert these changes since we follow a 1 function 1 PR policy for maintainability purposes

Reply:
I think the changes came when I merged changes from the upstream master, should I do the changes

Reply:
Yes please :)